2-mandatory/2-water-bottle.js

```diff
@@ -19,17 +19,22 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100;
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume < 100) this.volume += 10;
```

Comment:
I think because of the water bottle capacity this.volume < 100 should change to this.volume <= 90.